packages/components-angular/src/app/examples/modal-example-accessibility.component.ts

```diff
@@ -4,10 +4,10 @@ import { ChangeDetectionStrategy, Component } from '@angular/core';
   selector: 'page-modal-example-accessibility',
   template: `
     <p-button type="button" [aria]="{ 'aria-haspopup': 'dialog' }" (click)="onOpen()">Open Modal</p-button>
-    <p-modal [open]="isModalOpen" (dismiss)="onDismiss()">
+    <p-modal [open]="isModalOpen" (dismiss)="onDismiss()" [aria]="{ 'aria-label': 'Some Heading' }">
       <p-text>Some Content</p-text>
       <p-button-group class="footer">
```

Comment:
Do any styles exist for the `.footer` class here?
Also verify with all 4 framework implementations. Same applies for `type="button"` which still is missing in vanilla js and therefore not consistent with the other examples.